docs/reference/3-admin-api/installation/admin-api-1x-for-odsapi-5x-6x/admin-api-1x-docker-installation.md

```diff
@@ -0,0 +1,184 @@
+ADMIN_API_VIRTUAL_NAME=<virtual name for the Admin API endpoint>
+ODS_VIRTUAL_NAME=<virtual name for the ods endpoint>
+
+# For Authentication
+AUTHORITY=<Authentication Authority Appsetting Eg. http://localhost/${ADMIN_API_VIRTUAL_NAME}>
+ISSUER_URL=<Authentication IssuerUrl Appsetting Eg. https://localhost/${ADMIN_API_VIRTUAL_NAME}>
+SIGNING_KEY=<Authentication Signing Key (Symmetric Security Key) for Auth Tokens>
+
+# For Postgres only
+POSTGRES_USER=<default postgres database user>
+POSTGRES_PASSWORD=<password for default postgres user>
+PGBOUNCER_LISTEN_PORT=<port for pg bouncer to listen to>
+
+# The following needs to be set to specify a health check test for Admin api.
+# RECOMMENDED: To use the default internal Admin Api health check endpoint, set the variable as follows:
+ADMIN_API_HEALTHCHECK_TEST="curl -f http://${ADMIN_API_VIRTUAL_NAME}/health"
+#  To disable the health check, remove the above and instead set the variable as follows:
+# ADMIN_API_HEALTHCHECK_TEST=/bin/true
+#  To add a custom health check, consult the documentation at https://docs.docker.com/compose/compose-file/compose-file-v3/#healthcheck
+
+# The following needs to be set to specify the ODS API endpoint for Admin API to internally connect.
+# If user chooses direct connection between ODS API and Admin API within docker network, then set the api internal url as follows
+API_INTERNAL_URL = http://${ODS_VIRTUAL_NAME}
+```
+
+### Nginx / Gateway Configuration
+
+Update your nginx server configuration to include the Admin API in the reverse proxy.
+
+**default.conf.template**
+
+```
+# upstream server config...
+
+server {
+    #listen / server config...
+    #ssl_certificate config...
+
+    # other locations...
+
+    location /${ADMIN_API_VIRTUAL_NAME} {
+        client_max_body_size 20M;
+        proxy_pass http://${ADMIN_API_VIRTUAL_NAME};
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_set_header X-Forwarded-Host $host;
+        proxy_set_header X-Forwarded-Port 443;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    }
+}
+```
+
+## 2\. Relaunch the Docker Composition
+
+After updating the files, restart the docker composition.
+
+```
+docker compose -f ./compose/your-compose-file.yml --env-file ./.env up -d
+```
+
+## 3\. Execute First-Time Configuration
+
+Continue on to [First-Time Configuration for Admin 1.x](../admin-api-1x-for-odsapi-5x-6x/first-time-configuration-for-admin-api-1x.md).
+
+> [!NOTE]
+> The following is the DockerHub repo for **Admin API v1.4.0 Docker Image** for inclusion in Docker compose:
+>
+> * [edfialliance/ods-admin-api:v1.4.0](https://hub.docker.com/layers/edfialliance/ods-admin-api/v1.4/images/sha256-0a52face1b03e94892dc4d82e05f2fae05e635f1c46b2baf081bbcf2e81d76b1?context=explore)
+>
+> * [edfialliance/ods-admin-api-db:v1.4.0](https://hub.docker.com/layers/edfialliance/ods-admin-api-db/v1.4/images/sha256-70375e3564e9d409dfe8c25d27d504f1b3e15f75e454c9da9f5dc40c30c9d4a3?context=explore)
```

Comment:
Should change this to a [docusaurus-style admonition](https://docusaurus.io/docs/3.5.2/markdown-features/admonitions). It will not render the same as in GitHub.
```suggestion
::::note

The following is the DockerHub repo for **Admin API v1.4.0 Docker Image** for inclusion in Docker compose:

* [edfialliance/ods-admin-api:v1.4.0](https://hub.docker.com/layers/edfialliance/ods-admin-api/v1.4/images/sha256-0a52face1b03e94892dc4d82e05f2fae05e635f1c46b2baf081bbcf2e81d76b1?context=explore)
* [edfialliance/ods-admin-api-db:v1.4.0](https://hub.docker.com/layers/edfialliance/ods-admin-api-db/v1.4/images/sha256-70375e3564e9d409dfe8c25d27d504f1b3e15f75e454c9da9f5dc40c30c9d4a3?context=explore)

:::
```